src/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
why do we need another one license here?

Reply:
agree

we store all licenses in https://github.com/openvinotoolkit/openvino/tree/master/licensing

Reply:
never mind, those changes should be reverted in this PR (done)